frontend/src/components/RSTSVGLogo/RSTSVGLogo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import BCLogo from '@/images/BC_nav_logo.svg';
+import RSTLogo from '@/images/RST_nav_logo.svg';
+import '@/components/RSTSVGLogo/RSTSVGLogo.scss';
+
+export const RSTSVGLogo: React.FC = () => {
+  return (
+    <div
+      className={
+        'rst-svg-logo-container d-flex align-items-center justify-content-center px-4 w-100 h-100'
+      }
```

Comment:
```suggestion
      className="rst-svg-logo-container d-flex align-items-center justify-content-center px-4 w-100 h-100"
```

Total nit 😅 : we don't need the brackets if there are no dynamic classes. 

For your consideration: why inline so many bootstrap classes when we have a custom `rst-svg-logo-container` to add these into? Why add nested classes for the `.bc-logo`, `.rst-logo` widths instead of inline them as well?

Reply:
- imo using the bootstrap classes is better than writing extra css code

- nested classes were to keep them consistent with the parent where any custom styles are applied through scss files